content/en/agent/proxy.md

```diff
@@ -235,7 +251,7 @@ backend datadog-logs
     mode tcp
     option tcplog
```

Comment:
Starting with haproxy 1.8 the `tcplog` option is not supported inside backend section, you may move it into the relevant frontend section.